### DISCUSSION TITLE
Eurorack Power Supply

### DISCUSSION THREAD

**Discussion by drjay701** [DatanoiseTV/PicoDSP-Hardware#8] [Q&A]
Hi all,

I'd like to connect the board in a custom Eurorack module. Does the board include the Schottky diode / P-FET in order to connect USB and an external power supply at the same time or do I need to add these components?
Can external R2R Opamps be powered from the 3V3 Pin, or should I provide an additional 3V3 power regulator ?
BTW : is the 3V3 output pin vonnrctrd to the analog or digital LDO?

thanks for your help

**Comment by robinmayol** [comment]
Happy anniversary to this question! I am finding myself in a similar situation. I am powering the PicoADK via VSYS (as recommended for a Pico, to isolate the ext. power and USB power) but I am not getting any sound out. Is the DAC only powered via VBUS?
